packages/jest-cli/src/__tests__/runJest-test.js

```diff
@@ -1,104 +0,0 @@
-/**
```

Comment:
I removed this file in a later commit (not yet sent out) already where I fix up how updateConfig works. At that point, we won't be passing updateConfig to ProjectConfig any more and this test will not be useful any more. I have new tests that make sure this works in a follow-up.